package.json

```
@@ -50,6 +50,35 @@
         ],
         "default": "default"
       },
+      {
+        "name": "Token name format",
+        "type": "picker",
+        "id": "tokenNameFormat",
+        "submenu": true,
+        "options": [
+          {
+            "name": "Constant Case",
+            "value": "constant"
+          },
+          {
+            "name": "Snake Case",
+            "value": "snake"
+          },
+          {
+            "name": "Camel Case",
+            "value": "camel"
+          },
+          {
+            "name": "Pascal Case",
+            "value": "pascal"
+          },
+          {
+            "name": "Do not format",
+            "value": "none"
+          }
```

It might be good to illustrate what each option would look like in the option labels.

```suggestion
          {
            "name": "CONSTANT_CASE",
            "value": "constant"
          },
          {
            "name": "snake_case",
            "value": "snake"
          },
          {
            "name": "camelCase",
            "value": "camel"
          },
          {
            "name": "PascalCase",
            "value": "pascal"
          },
          {
            "name": "Use original",
            "value": "none"
          }
```